indicators/models.py

```diff
@@ -1,40 +1,40 @@
 from django.db import models
-from django.core.exceptions import ValidationError
-from django.core.validators import MaxValueValidator, MinLengthValidator
+from django.core.validators import MaxValueValidator, MinValueValidator,\
+                                   MinLengthValidator, MaxLengthValidator
 
 
 MAX_FIELD_REPRESENTATION = 20
 
 
 class CountryIndicator(models.Model):
-    iso_code = models.CharField(max_length=3, primary_key=True,
-                                validators=[MinLengthValidator(3)])
-    name = models.CharField(max_length=50, unique=True)
-    PDI = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
-    IND = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
-    MAS = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
-    UAI = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
-    LTO = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
-    IVR = models.PositiveIntegerField(validators=[MaxValueValidator(100)])
+    iso_code = models.CharField(max_length=3, unique=True,
+                                validators=[MinLengthValidator(3),
+                                            MaxLengthValidator(3)])
+    name = models.CharField(max_length=200, unique=True)
+    pdi = models.PositiveIntegerField(verbose_name='PDI',
+                                      validators=[MinValueValidator(0),
+                                                  MaxValueValidator(100)])
+    ind = models.PositiveIntegerField(verbose_name='IND',
+                                      validators=[MinValueValidator(0),
+                                                  MaxValueValidator(100)])
+    mas = models.PositiveIntegerField(verbose_name='MAS',
+                                      validators=[MinValueValidator(0),
+                                                  MaxValueValidator(100)])
+    uai = models.PositiveIntegerField(verbose_name='UAI',
+                                      validators=[MinValueValidator(0),
+                                                  MaxValueValidator(100)])
+    lto = models.PositiveIntegerField(verbose_name='LTO',
+                                      validators=[MinValueValidator(0),
+                                                  MaxValueValidator(100)])
+    ivr = models.PositiveIntegerField(verbose_name='IVR',
```

Comment:
According to [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) I'd propose to define custom field which encapsulates all boilerplate code. E.g. something like next one:
```python
class IndicatorField(PositiveIntegerField):
    default_validators = [MinValueValidator(0), MaxValueValidator(100)]
```
Then you may write something like this one:
```python
class CountryIndicator(models.Model):
    # .... something
    ind = IndicatorField(verbose_name='IND')
    mas = IndicatorField(verbose_name='MAS')
    # ... etc.
```